src/App.js

```diff
@@ -1,13 +1,34 @@
 import React from "react";
-
+import Heading from "./Heading";
+//import Search from "./Search";
+import TouristInfoCard from "./TouristInfoCards";
 import Bookings from "./Bookings";
+import Restaurant from "./Restaurant";
+import Footer from "./Footer";
 import "./App.css";
 
+
 const App = () => {
+  const contactDetails = [
+    "123 Fake Street, London, E1 4UD",
+    "hello@fakehotel.com",
+    "0123 456789"
+  ];
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <Heading />
+      <div className='cards-container'>
+          <TouristInfoCard image="Glasgow.jpeg" title="Glasgow" description="Glasgow is a port city on the River Clyde in Scotland's western
+        Lowlands. It's famed for its Victorian and art nouveau architecture..." link="https://peoplemakeglasgow.com"/> 
+          <TouristInfoCard image="London.jpg" title="London" description="London, the capital of England and the United Kingdom, is a
+         21st-century city with history stretching..." link="https://visitlondon.com"/> 
+          <TouristInfoCard image="Manchester.jpg" title="Manchester" description="Manchester is a major city in the northwest of England with 
```

Comment:
Nice laxmi!!, you have really understood using components to remove duplication!! You are the first ive seen to use a separate card component for each card here 😄

Reply:
thank you sam for your beautiful comment.